### ISSUE TITLE
startsWith() on null object at ScreenEncoder.java:158

### ISSUE THREAD

**Issue by SwayChen** [Genymobile/scrcpy#365]
Model: Meizu 16th
OS: Flyme 7.1.1.2A base on Android 8.1.0

log like this: 
```
MacBook-Pro:~ allconnected$ scrcpy
/usr/local/Cellar/scrcpy/1.5/share/scrcpy/scrcpy-server.jar: 1 file pushed. 1.2 MB/s (19308 bytes in 0.016s)
ERROR: Exception on thread Thread[main,5,main]
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.startsWith(java.lang.String)' on a null object reference
	at android.media.MediaCodec.configure(MediaCodec.java:1918)
	at android.media.MediaCodec.configure(MediaCodec.java:1873)
	at com.genymobile.scrcpy.ScreenEncoder.configure(ScreenEncoder.java:158)
	at com.genymobile.scrcpy.ScreenEncoder.streamScreen(ScreenEncoder.java:68)
	at com.genymobile.scrcpy.Server.scrcpy(Server.java:25)
	at com.genymobile.scrcpy.Server.main(Server.java:109)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:285)
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
2018-12-04 10:17:42.744 scrcpy[1111:16013] ERROR: Could not open video stream
2018-12-04 10:17:42.789 scrcpy[1111:15936] INFO: Initial texture: 1080x2160
```

**Comment by thinkhy** [comment]
Refer for the blog for a workable fix: https://www.jianshu.com/p/0d29cc53199c

**Comment by SwayChen** [comment]
Thanks for temporary solution by @thinkhy

**Comment by Jingle-seven** [comment]
meizu biss

**Comment by rom1v** [comment]
See https://github.com/Genymobile/scrcpy/pull/888

**Comment by rom1v** [comment]
Could you please replace [`scrcpy-server.jar`](https://tmp.rom1v.com/scrcpy/issue365/1/scrcpy-server.jar) in your scrcpy v1.10 and test again?
_SHA256: b34c8ce7f8f60e6d6ef42c3b0975c6b7650755f2d50205a2e950f996e2091bbc_

**Comment by SwayChen** [comment]
@rom1v Yep, it works fine with v1.10.

![Scrcpy Version](https://user-images.githubusercontent.com/6345749/68915553-86bcc700-079d-11ea-94be-10b0d1e01d98.png)
![Meizu Device Info](https://user-images.githubusercontent.com/6345749/68915679-0e0a3a80-079e-11ea-904a-fa8b93cf4f76.png)

**Comment by rom1v** [comment]
Thank you for your feedbacks, I'll merge it for v1.11 :+1: (the version number is still 1.10 but it's not the 1.10 release, it's a dev version).

**Comment by rom1v** [comment]
To fix #994, I changed the way the workaround is called.

Could you please take scrcpy v1.13 and replace this file:

[`scrcpy-server`](https://tmp.rom1v.com/scrcpy/issue994/1/scrcpy-server)
_SHA256: b64e6672eeb2b7c00c6f83673783bd8839ba65ea2ac7edcdeeca44676047328b_

and confirm that it works without the error message.

Thank you.
